include/bluetooth/bluetooth.h

```diff
@@ -1852,50 +1855,65 @@ void bt_le_scan_cb_register(struct bt_le_scan_cb *cb);
 void bt_le_scan_cb_unregister(struct bt_le_scan_cb *cb);
 
 /**
- * @brief Add device (LE) to whitelist.
+ * @brief Add device (LE) to filter accept list.
  *
- * Add peer device LE address to the whitelist.
+ * Add peer device LE address to the filter accept list.
  *
- * @note The whitelist cannot be modified when an LE role is using
- * the whitelist, i.e advertiser or scanner using a whitelist or automatic
- * connecting to devices using whitelist.
+ * @note The filter accept list cannot be modified when an LE role is using
+ * the filter accept list, i.e advertiser or scanner using a filter accept list
+ * or automatic connecting to devices using filter accept list.
  *
  * @param addr Bluetooth LE identity address.
  *
  * @return Zero on success or error code otherwise, positive in case of
  *         protocol error or negative (POSIX) in case of stack internal error.
  */
-int bt_le_whitelist_add(const bt_addr_le_t *addr);
+int bt_le_fa_list_add(const bt_addr_le_t *addr);
+__deprecated
+static inline int bt_le_whitelist_add(const bt_addr_le_t *addr)
+{
+	return bt_le_fa_list_add(addr);
+}
 
 /**
- * @brief Remove device (LE) from whitelist.
+ * @brief Remove device (LE) from filter accept list.
  *
- * Remove peer device LE address from the whitelist.
+ * Remove peer device LE address from the filter accept list.
  *
- * @note The whitelist cannot be modified when an LE role is using
- * the whitelist, i.e advertiser or scanner using a whitelist or automatic
- * connecting to devices using whitelist.
+ * @note The filter accept list cannot be modified when an LE role is using
+ * the filter accept list, i.e advertiser or scanner using a filter accept list
+ * or automatic connecting to devices using filter accept list.
  *
  * @param addr Bluetooth LE identity address.
  *
  * @return Zero on success or error code otherwise, positive in case of
  *         protocol error or negative (POSIX) in case of stack internal error.
  */
-int bt_le_whitelist_rem(const bt_addr_le_t *addr);
+int bt_le_fa_list_rem(const bt_addr_le_t *addr);
```

Comment:
Using mixed abbreviation of `fa` and `FILTER_ACCEPT_LIST` in scan options, and `FILTER` in other places.

`bt_le_filter_accept_list_add` and friends does not seem so bad to me, so I would suggest that instead.

Reply:
OK, fair enough. `bt_le_filter_accept_list_remove` is going to be a mouthful though ;)